chart/flux/templates/deployment.yaml

```diff
@@ -252,6 +252,9 @@ spec:
           {{- if .Values.token }}
```

Comment:
I do not think this should be nested, can you think of a way that both values can be used (`token` / `connect`), while still making it possible to connect to the `cloud.weave.works` endpoints when only a token is given for backwards compatibility?